src/ScottPlot/Plottable/RadarPlot.cs

```diff
@@ -10,21 +11,22 @@ public class RadarPlot : IPlottable
     {
         private readonly double[,] Norm;
         private readonly double NormMax;
-        private readonly double[] NormMaxes;
-        public string[] CategoryLabels;
-        public string[] GroupLabels;
+        private readonly double[]? NormMaxes;
+        public string[]? CategoryLabels;
+        public string[]? GroupLabels;
         public Color[] FillColors;
         public Color[] LineColors;
         public Color WebColor = Color.Gray;
-        public readonly bool IndependentAxes;
+        [MemberNotNull(nameof(NormMaxes))]
+        public bool IndependentAxes { get; }
```

Comment:
Converted from read-only field to get-only property. Binary breaking change. Recompilation should work